src/ccnl-core/include/ccnl-pkt-util.h

```diff
@@ -22,8 +22,9 @@
 #ifndef CCNL_PKT_UTIL_H
 #define CCNL_PKT_UTIL_H
 
-#include <stddef.h>
+#include <stdint.h>
 #include <stdbool.h>
+#include <stddef.h>
 
 bool
 ccnl_isSuite(int suite);
```

Comment:
int?

Reply:
why? ``bool`` was introduced as a type in C99

Reply:
currently we have kernel support, therefore there is no support for C99